packages/main-ui/src/pages/components/user_setup_popup.rs

```diff
@@ -62,49 +63,67 @@ pub fn UserSetupPopup(
                 }
 
                 // Nickname
-                div { class: "flex flex-col gap-[8px]",
-                    div { class: "flex flex-row items-start",
-                        span { class: "text-[14px] font-bold leading-[24px]", "닉네임" }
-                        span { class: "text-[14px] text-[#ff0000]", "*" }
-                    }
-                    div { class: "flex flex-col items-start w-full mt-[10px] gap-[8px]",
+                div { class: "grid gap-8 w-full grid-cols-2",
+                    div { class: "flex flex-col items-start gap-5",
+                        span { class: "text-neutral-400 text-[15px]/28 font-bold", "{tr.first_name}" }
                         input {
-                            class: "w-[400px] max-[400px]:w-[300px] h-[59px] px-[24px] py-[17.5px] bg-[{theme.background}] text-[18px] font-bold leading-[24px] rounded-[4px] placeholder-[{theme.primary07}] rounded-[8px]",
+                            class: "max-[400px]:w-[300px] h-[59px] px-[24px] py-[17.5px] bg-[{theme.background}] text-[18px] font-bold leading-[24px] placeholder-[{theme.primary07}] rounded-[8px]",
                             placeholder: "{tr.enter_nickname}",
-                            value: nickname(),
+                            value: firstname(),
                             oninput: move |e| {
                                 let value = e.value();
-                                valid.set(value.chars().all(|c| c.is_alphanumeric()));
-                                nickname.set(value);
+                                fname_valid.set(value.chars().all(|c| c.is_alphanumeric()));
+                                firstname.set(value);
                             },
+                            if !fname_valid() {
+                                span { class: "text-[14px] font-bold leading-[24px] text-[{theme.primary04}]",
+                                    "{tr.special_characters}"
+                                }
+                            }
                         }
-                        if !valid() {
-                            span { class: "text-[14px] font-bold leading-[24px] text-[{theme.primary04}]",
-                                "{tr.special_characters}"
+                    }
+                    div { class: "flex flex-col items-start gap-5",
+                        span { class: "text-neutral-400 text-[15px]/28 font-bold", "{tr.last_name}" }
+                        input {
+                            class: "max-[400px]:w-[300px] h-[59px] px-[24px] py-[17.5px] bg-[{theme.background}] text-[18px] font-bold leading-[24px] placeholder-[{theme.primary07}] rounded-[8px]",
```

Comment:
don't use variable in tailwindcss class. tailwindcss compiler won't compile properly.

As I share in our meeting, use placeholder-primary with predefined `input.css`